service/src/battery.cpp

```diff
@@ -33,6 +33,7 @@ Battery::Battery(Logger* newLogger, bool loglevelSet, QObject *parent) : QObject
     updateTimer = new QTimer(this);
     highNotifyTimer = new QTimer(this);
     lowNotifyTimer = new QTimer(this);
+    healthNotifyTimer = new QTimer(this);
```

Comment:
The timers are *really* inconsistent when the device is in deep sleep, so these will most likely get redone - released in version 4 I think...